demos/src/demo-brush.js

```diff
@@ -29,16 +31,20 @@ function createBrushChart() {
                 d3Selection.select('.js-start-date').text(format(brushExtent[0]));
                 d3Selection.select('.js-end-date').text(format(brushExtent[1]));
 
-                d3Selection.select('.js-date-range').classed('is-hidden', false);
+                elementDateRange.classed('is-hidden', false);
             })
             .on('customBrushEnd', function(brushExtent) {
                 // eslint-disable-next-line no-console
                 console.log('rounded extent', brushExtent);
+
+                if (null === brushExtent[0]) {
```

Comment:
Is there a reason why we can't do:
```if (!brushExtent[0]) { ... }```

Reply:
No, we could do that. I was just being explicit; this should only ever be `null` or a `Date`.

Reply:
I would just reverse this to read:
   if (brushExtent[0] === null) ...